### PR TITLE
Update deploy-on-aws-eks.md (#1021)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #1021 to release-1.1
You can switch your code base to this Pull Request by using [git-extras](https://github.com/tj/git-extras):
```bash
# In docs-tidb-operator repo:
git pr https://github.com/pingcap/docs-tidb-operator/pull/1040
```

After apply modifications, you can push your change to this PR via:
```bash
git push git@github.com:ti-srebot/docs-tidb-operator.git pr/1040:release-1.1-21375ff056a1
```

---

Added reference link for tidb-cluster CR and API

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/deploy-on-aws-eks.md [hunk 1]
```diff
@@ -168,13 +168,29 @@ kubectl create namespace tidb-cluster
...
+```
+
+Refer to [configure the TiDB cluster](configure-a-tidb-cluster.md) to further customize and configure the CR before applying.
+
 To deploy the `TidbCluster` and `TidbMonitor` CR in the EKS cluster, run the following command:
 
 {{< copyable "shell-regular" >}}
 
 ```shell
+<<<<<<< HEAD
```
Comment (ran-huang) [inline en/deploy-on-aws-eks.md:187]: @DanielZhangQD PTAL

File: en/deploy-on-aws-eks.md [hunk 2]
```diff
@@ -168,13 +168,29 @@ kubectl create namespace tidb-cluster
...
 kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.1.6/examples/aws/tidb-cluster.yaml -n tidb-cluster && \
 kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.1.6/examples/aws/tidb-monitor.yaml -n tidb-cluster
+=======
+kubectl apply -f tidb-cluster.yaml -n tidb-cluster && \
+kubectl apply -f tidb-monitor.yaml -n tidb-cluster
+>>>>>>> 21375ff... Update deploy-on-aws-eks.md (#1021)
```
Comment (ran-huang) [inline en/deploy-on-aws-eks.md:193]:
```suggestion
kubectl apply -f tidb-cluster.yaml -n tidb-cluster && \
kubectl apply -f tidb-monitor.yaml -n tidb-cluster
```

**Review by ran-huang — APPROVED** [review]

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Pull request merged**